src/scenes/CollectionCreationFlow/steps/AddNfts/Editor/StagedNftWrapper.tsx

```diff
@@ -0,0 +1,41 @@
+import { Nft } from 'types/Nft';
+import SortableStagedNft from './SortableStagedNft';
+import UnstageButton from './UnstageButton';
+import styled from 'styled-components';
+import { StyledNftPreviewLabel } from 'components/NftPreview/NftPreviewLabel';
+
+type Props = {
+  nft: Nft;
+  activeId?: string;
+};
+
+function StagedNftWrapper({ nft, activeId }: Props) {
+  return (
+    <StyledStageNftWrapper>
+      <StyledUnstageButton
+        // TODO make index guaranteed
+        nftIndex={nft.index || 0}
```

Comment:
I think we should differentiate between the existing `Nft` type and an "editable" `Nft` type that has fields specific to the context of the Collection Editor, like `index` and `isSelected`

Reply:
or wrap the `Nft` type in a new object that looks like 
{
nft: Nft
index: number
isSelected: boolean
}
that's probably better

Reply:
what do you think @Robinnnnn

Reply:
makes total sense!